EIPS/eip-1080.md

```diff
@@ -0,0 +1,185 @@
+---
+eip: 1080
+title: Recoverable Token
+author: Bradley Leatherwood <bradleat@inkibra.com>
+discussions-to: <bradleat@inkibra.com>
```

Comment:
Please provide a URL where people can discuss publicly, such as a thread on ethereum-magicians, or an issue in the EIPs repo.